README.md

```diff
@@ -1 +1 @@
-https:///psu.github.i0
```

Comment:
<!DOCTYPE html>

<html >

<head>
    <meta charset="UTF-8">
    <title>PSU Homepage</title>
    <style>
        body{
             font-family: arial;
            background: #000000d9;
            color: white;
            text-align: center;
        }
        .welcome{
            color: blue;
            font-size: 27px;
            font-weight: 
        }
        .top span{
            color: yellow;
        }
        .info{
            border: 2px solid violet;
            padding: 20px;
            margin: 10px;
            text-align: left;
        }
        b{
            color: pink;
            font-size: 20px
        }
        img{
            width: 100%;
        }
        a{
            color: white;
        }
    </style>
</head>
<body>
    <center>
      <br>
       <h1 class="welcome">Welcome To PSU</h1>
       <h3 class="top">Pangasinan State University-<span>Lingayen</span></h3>
       <br>
       <br><img src="imges.jpeg">
        <h1>About PSU</h1>
        <hr>
        <p>Pangasinan State University (Filipino: Pamantasang Pampamahalaan ng Pangasinan) also referred to by its acronym PSU or (PangSU) is a public university in Pangasinan province, Philippines. The university was founded in its current form in 1979, although its origins trace back to the 1920s. PSU is notable for its many locations throughout the province of Pangasinan. It is mandated to provide advanced instruction in the arts, agricultural and natural sciences as well as in technological and professional fields. Its main campus is located in Lingayen, Pangasinan. Other campuses are located in Alaminos, Asingan, Bayambang, Binmaley, Infanta, San Carlos City, Santa Maria, and Urdaneta City. The PSU School of Advanced Studies (SAS) is located in Urdaneta City and the Open University Systems (OUS) is located in Lingayen Campus. </p>
        <br>
        <br><b>Website:</b> www.psu.edu.ph <br> <b>Admission :</b> <a href =https://admission.psu.edu.ph />click here</a> </div> <h1><a href="">Click Here to Visit PSU website</a></h1> <br> <br> <!-- Designed my Webbyimros <div class="copyright"> <p>Code Idea By <a href="https://www.youtube.com /channel/UC-KSCI0RPF9fI1U G269b4pg">Webbylmros</a>< /p> </div> </center> </body> </html>